spec/models/journals_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+RSpec.describe Journal, type: :model do
+  let(:journal) { FactoryBot.create(:journal) }
+
+  it 'checks if Journal can be created with all variables' do
+    expect(journal).to be_valid
+  end
+
+  it 'validates presence of attributes' do
+    should validate_presence_of(:title).with_message(I18n.t('validations.presence'))
+    should validate_presence_of(:date).with_message(I18n.t('validations.presence'))
+    should validate_presence_of(:content).with_message(I18n.t('validations.presence'))
+    should validate_presence_of(:goals_today).with_message(I18n.t('validations.presence'))
+    should validate_presence_of(:goals_tomorrow).with_message(I18n.t('validations.presence'))
+  end
```

Comment:
Let's follow this review: https://github.com/meb-wtag/journey_me/pull/5#discussion_r1342662327